python/lsst/faro/measurement/MatchedCatalogMeasurementTasks.py

```diff
@@ -96,8 +96,8 @@ class PF1TaskConfig(Config):
                          dtype=float, default=np.Inf)
     nMinPhotRepeat = Field(doc="Minimum number of objects required for photometric repeatability.",
                            dtype=int, default=50)
-    # The defaults for threshPA2 and threshPF1 correspond to the SRD "design" thresholds.
-    threshPA2 = Field(doc="Threshold in mmag for PF1 calculation.", dtype=float, default=15.0)
+    # The defaults for threshPA2 correspond to the SRD "design" thresholds.
+    threshPA2 = Field(doc="Threshold in mmag for PA2 calculation.", dtype=float, default=15.0)
```

Comment:
Actually, the threshold is _called_ PA2, but it is a threshold for calculation of PF1. I think the previous text in this docstring was OK. [Removing threshPF1 from line 99 makes sense, though.]